services/graphite/service.go

```diff
@@ -334,6 +335,13 @@ func (s *Service) handleLine(line string) {
 	// Parse it.
 	point, err := s.parser.Parse(line)
 	if err != nil {
+		switch err := err.(type) {
+		case *ErrUnsupportedValue:
+			// Graphite ignores NaN values with no error.
+			if math.IsNaN(err.Value) {
```

Comment:
I added it as `statPointsNaNFail` (as NaN is usually capitalized that way instead of Nan). If you want the other capitalization, I can amend the commit again.